samples/snippets/src/main/java/com/example/monitoring/CreateAlertPolicy.java

```diff
@@ -0,0 +1,96 @@
+import com.google.monitoring.v3.ProjectName;
+import com.google.protobuf.Duration;
+import java.io.IOException;
+
+// Sample to create an alert policy
+public class CreateAlertPolicy {
+
+  public static void main(String[] args) throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "your-project-id";
+    String alertPolicyName = "your-policy-name";
+    createAlertPolicy(projectId, alertPolicyName);
+  }
+
+  public static void createAlertPolicy(String projectId, String alertPolicyName)
+      throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests.
+    try (AlertPolicyServiceClient alertPolicyServiceClient = AlertPolicyServiceClient.create()) {
+      ProjectName name = ProjectName.of(projectId);
+
+      // A Filter that identifies which time series should be compared with the threshold
+      String metricFilter =
+          "metric.type=\"compute.googleapis.com/instance/"
+              + "cpu/utilization\" AND resource.type=\"gce_instance\"";
+
+      // Build Duration
+      Duration aggregationDuration = Duration.newBuilder().setSeconds(60).build();
+
+      // Build Aggregation
+      Aggregation aggregation =
+          Aggregation.newBuilder()
+              .setAlignmentPeriod(aggregationDuration)
+              .setCrossSeriesReducer(Aggregation.Reducer.REDUCE_MEAN)
+              .setPerSeriesAligner(Aggregation.Aligner.ALIGN_MAX)
+              .build();
+
+      // Build MetricThreshold
+      AlertPolicy.Condition.MetricThreshold metricThreshold =
+          AlertPolicy.Condition.MetricThreshold.newBuilder()
+              .setComparison(ComparisonType.COMPARISON_GT)
+              .addAggregations(aggregation)
+              .setFilter(metricFilter)
+              .setDuration(aggregationDuration)
+              .build();
+
+      // Construct Condition object
+      AlertPolicy.Condition alertPolicyCondition =
+          AlertPolicy.Condition.newBuilder()
+              .setDisplayName(alertPolicyName)
+              .setConditionThreshold(metricThreshold)
+              .build();
+
+      // Build an alert policy
+      AlertPolicy alertPolicy =
+          AlertPolicy.newBuilder()
+              .setDisplayName(alertPolicyName)
+              .addConditions(alertPolicyCondition)
+              .setCombiner(AlertPolicy.ConditionCombinerType.AND)
+              .build();
+
+      // Create an alert policy
+      AlertPolicy actualAlertPolicy = alertPolicyServiceClient.createAlertPolicy(name, alertPolicy);
+      String alertPolicyId =
+          actualAlertPolicy.getName().substring(actualAlertPolicy.getName().lastIndexOf("/") + 1);
+      System.out.println("alert policy created:" + alertPolicyId);
+    } catch (ApiException ex) {
+      System.out.print("\nalert policy was not created." + ex.toString());
+    }
```

Comment:
\n as string constants get picked up by static analyzers.
If you aren't doing anything with an exception, you probably shouldn't be catching it.  
See our [sample format guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#exception-handling) for more info.

Reply:
Done